### DISCUSSION TITLE
BMS does not show up in Venus OS ?

### DISCUSSION THREAD

**Discussion by MartinSaaby** [Louisvdw/dbus-serialbattery#186] [Q&A] [answered]
Hi

I have just soldered the wires according to
https://github.com/Louisvdw/dbus-serialbattery/discussions/180

After that I ran the install without problems.
I edited the .../utils.py file on the venus.

But my BMS does not show up in Venus os?
Should I do something after reboot ?
(DVCC is enabled)

Note: I use a Daly 250A smart BMS with a Victron RS485 - USB cable.
I use ESS I don't know if that matters?

I am using the CAN/RS485 port on the BMS not UART. (I use UART for BT)
<img width="663" alt="Skærmbillede 2022-08-21 kl  14 08 18" src="https://user-images.githubusercontent.com/88431885/185790215-238fecdb-ad0a-4090-a217-85d47746750b.png">

![IMG_2393](https://user-images.githubusercontent.com/88431885/185788442-01c97f1b-419a-4b96-8703-b45286c1ff42.jpeg)
![IMG_2394](https://user-images.githubusercontent.com/88431885/185788446-f2b86bac-f16a-4cb6-91ec-5a24c6f62328.jpeg)
<img width="493" alt="Skærmbillede 2022-08-21 kl  13 16 10" src="https://user-images.githubusercontent.com/88431885/185788459-35c4b534-694a-46fc-af21-864920c827ae.png">

**Comment by Louisvdw** [comment, marked as answer]
The Victron cable is an isolated cable. 
You need to power it from both sides, so your red power wire need to be connected to 5V as well.

**Reply by MartinSaaby** [reply]
Thanks. How would you recommend I do that ? I can't see 5v in the connection?

I use the PX connection
<img width="841" alt="Skærmbillede 2022-08-21 kl  16 10 49" src="https://user-images.githubusercontent.com/88431885/185872816-c0b30fdf-a0a0-4e96-be67-761c61130786.png">

**Reply by Louisvdw** [reply]
It should be fine to use the POW_3.3V from either J6 or J4.

**Reply by MartinSaaby** [reply]
Ok. So I just connect the red wire to 3,3v (in J6 or J4) and leave the other cables (yellow, orange and black) connected to Px.
(GND is common?) 

Right?

**Reply by Louisvdw** [reply]
Yes exactly

**Reply by MartinSaaby** [reply]
Thanks. 
While measuring to ensure I got 3,3v I might have shorted some of the pins, så now the BMS has shut down.
I will try to see if I can get it up again 👍
